lib.php

```diff
@@ -88,6 +84,20 @@ function get_objectfs_config() {
     $config->logging = 0;
     $config->preferexternal = 0;
 
+    // TODO: load based on fs? move to client files?
+
+    // '\tool_objectfs\s3_file_system'
```

Comment:
Thoughts on splitting the definition of these object storage specific config variables out into a seperate object_settings (or another name) class . We could also move test_connection, test_permissions and define_client_section into this class to keep the object client classes more focused

Reply:
I would also namespace the config variable names to avoid collisions in the future. 

e.g. $config->aws_key

though this namespacing could be handled outside of the new object settings class if implemented.